test/commands/metric/create_test.rb

```diff
@@ -55,9 +55,9 @@ class Metric::CreateTest < ActiveSupport::TestCase
     action = :start_solution
     solution = create :concept_solution
     occurred_at = Time.current - 2.seconds
-    Exercism.request_context = { remote_ip: nil }
+    request_context = { remote_ip: nil }
```

Comment:
There is no need to set the global `Exercism.request_context` value in this test, and it might even have caused tests to fail due to the context changing whilst they are running. 🤞